src/plugins/vis_types/pie/public/pie_component.tsx

```diff
@@ -186,10 +187,10 @@ const PieComponent = (props: PieComponentProps) => {
   const { visData, visParams, services, syncColors } = props;
 
   function getSliceValue(d: Datum, metricColumn: DatatableColumn) {
-    if (typeof d[metricColumn.id] === 'number' && d[metricColumn.id] !== 0) {
+    if (typeof d[metricColumn.id] === 'number') {
```

Comment:
I'd use a `Number.isFinite` test instead of checking for a numeric type, it's slightly stricter as it excludes NaN and infinities